docs/students/working-through-a-lesson.md

```diff
@@ -1,5 +1,21 @@
-# Working through a Lesson
+# Working Through a Lesson
 
-This is Bible First. We’re all about lessons. It’s pretty easy - you just read it.
+### Step One: Read the Lesson
 
-Amet eligendi reprehenderit vitae eius et mollitia Quas aliquid quasi at reprehenderit officiis Minus dignissimos voluptatum maxime error aliquid. Ut ullam exercitationem a quae ipsam dolorem perspiciatis. Maiores voluptatibus excepturi.
+To read your lesson, [log in](https://biblefirst.online/en/sign_in) to your account and click the "Read" link at the top of the page.
+
+### Step Two: Take the Quiz
+
+To take the quiz, click the "Quiz" link at the top of the page and answer the questions. Once you have submitted the quiz, your coach will grade it, and you will receive an email once your quiz is ready for review.
+
+### Step Three: Review Your Quiz
+
+Click the link in the notification email to review your graded quiz. Keep an eye out for any comments from your coach, and don't hesitate to comment back with any questions you may have!
+
+### Step Four: Rinse and Repeat
+
+Now that your quiz has been graded, the next lesson is now available for reading!
+
+***
+\
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/CSIP8_vp95A" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
FWIW, I just watched this video you made. Excellent work on that! 🙌🏻  Sound was great, explanations clear. I'm impressed. 🤩